itest/lnd_funding_test.go

```diff
@@ -278,6 +278,10 @@ func basicChannelFundingTest(ht *lntest.HarnessTest,
 // testUnconfirmedChannelFunding tests that our unconfirmed change outputs can
 // be used to fund channels.
 func testUnconfirmedChannelFunding(ht *lntest.HarnessTest) {
+	if ht.IsNeutrinoBackend() {
```

Comment:
Are we planning on resurrecting this test after some of the other changes in the PR series?
I understand that it's a bit harder to set up for Neutrino since we don't see any unconfirmed balance.

But couldn't we send some coins to the wallet, confirm those, then do a self-transfer that is unconfirmed (that way even Neutrino nodes will see the TX as they've created it themselves), then open a channel from that?

I think it might be good to keep the test if possible since this scenario is one that seems to happen some times in the wild.

Reply:
I like it! Updated as suggested!

Reply:
I don't see a push of new commits since your comment?